# Structural and Dynamic Characterization of a Highly Flexible Protein by AlphaFold-Multimer and MD Simulations Integrating DEER and SAXS Data

## COMPUTATIONAL MODELING
Considering the spatiotemporal heterogeneity and high conformational flexibility of Thalassosira pseudonana CP12, we designed a three-stage modeling strategy. First, initial guess structures of the CP12 dimer were predicted by AlphaFold2.[@jumper2021;@evans2021]
Then, the most likely structure of them was roughly refined by the harmonic restrained all-atom molecular dynamics (MD) simulations so as to meet the experimental (SAXS[@shao2021] and EPR/DEER) data. Finally, the structures thus obtained were further refined by restrained-ensemble molecular dynamics (reMD) simulations[@roux2013;@shen2015;@islam2013;@qi2020] so that the distance distributions of all spin pairs agree between reMD simulations and the experimental EPR/DEER data.

![(A) Refinement scheme for the model obtained from AlphaFold2. (B) Starting model for the harmonic restrained MD simulations. C) Distances (blue) and distance distributions (black) between spin label pairs of the starting model for reMD simulation and DEER experiment data, respectively.](figures/cp12/scheme.jpg){#fig:scheme width=100%}

### FASTA sequence{#sec:sequence}
The full-length amino acid sequence including the 163 residues (red) that were used as input for AlphaFold2 modeling:
\newline MKIFLASLIGSCAAFAPAPFGKSPTALFGRVDTS\textcolor{blue}{AIEAALDASKKFGSTSSEARVLWDIVEEMD
ASDNSVASKAPIVDSEYEAKVKSLSQMLTKTKAELDQVKALADDLKGVKLASPSVGSSAP
DDSVMKEALAAARAATEEFGQSSPQARLAWETVEEIAASPVDIRAPLDEECLIELIEGCEA
LEKFQAALGSR}

### Starting model
Structure prediction for the CP12 homodimer was performed using AlphaFold v2.1.1-Multimer (AF2) and the default databases[@jumper2021;@evans2021;@tunyasuvunakool2021]. The amino acid sequence of the wild-type (WT) was downloaded from NCBI[@shao2021]. A truncated FASTA sequence containing 163 residues (the blue part of sequence in @sec:sequence) without His-tag was used in the AF2 input file for multiple sequence alignments (MSAs) lookup and structural template matching. For WT, S46C and S56C mutants, their homodimeric structure was predicted with the top five ranked models. For the model confidence, along with the prediction results, a measure called predicted local distance difference test score (pLDDT, on a scale from 0 - 100, where 100 represents the most confident) was used to estimate the per-residue confidence.

### Refinement by Molecular Dynamics Simulations 
To refine the starting model thus obtained, all-atom harmonic restrained MD simulations without spin labels followed by restrained-ensemble MD (reMD) simulations with spin labels were performed to obtain a realistic conformational ensemble that fit with both the experimental SAXS curve[@shao2021] and the spin-spin distance distributions derived from EPR/DEER experiments. The simulation process is summarized in a logic diagram, shown in @fig:scheme A. Due to the high flexibility of R1 spin labels with 5 dihedral angles, the reMD simulation[@roux2013] technique for the model with all-atom spin labels is suitable to exploit protein’s structure in their native environment based on multiple distance histograms information obtained from EPR/DEER spectroscopy. It should be noted that a large discrepancy of the spin pair distance in the initial structure of reMD simulation with DEER data may cause extraordinary rapid atomic movements leading to unexpected termination during the simulation process. Thus, to avoid such violation, a screening on the starting model is needed based on the distance information between all spin pairs, before each run of reMD simulation.

We started the structural refinement using MD simulations from a model (@fig:scheme B) that was modified from the AF2 model (S56-2 of @fig:af2-output). Despite their high conformational flexibility that was revealed by DEER (C150 in @fig:scheme C), these two well-structured C-terminal helices were overfolded with the coiled-coil in the AF2 model (@fig:predicted-model) and could not be efficiently sampled by our preliminary calculations with conventional constant temperature, constant pressure MD simulations, even at a temperature higher than room temperature. Therefore, we manually moved two C-terminal helices away from the coiled-coil part of the dimer (@fig:scheme B). The computational detail of the subsequent simulations is explained below.

![Top 5 homodimer models for WT, S46C and S56C mutants are shown with each residue being colored according to its per-residue confidence score (pLDDT): Blue (high), cyan (high medium), yellow (low medium) and red (low).](figures/cp12/af2-output.jpg){#fig:af2-output width=100%}

(1)	We conducted harmonic restrained MD simulations with the Amber 20 package[@case2020] based on the modified AF2 model. The Amber ff19SB force field[@tian2020] was used for the protein, which was immersed into a cubic periodic box filled with water solvent molecules modeled by the OPC model[@izadi2014] with the LEaP program of AmberTools20. All charged residues were considered in their standard protonation state at pH = 7.0. We added 28 sodium ions to neutralize the simulation box and the total number of atoms became 178352. Nonbonded particle-particle interactions[@duan2001a] were considered with a distance cutoff of 9 Å and the long-range electrostatic interactions were treated with the particle mesh Ewald (PME) method[@salomon-ferrer2013].
After minimization, heating, and equilibration of the simulation system, several rounds of MD simulations were conducted imposing harmonic restraints on the CA-CA distances across the dimer for respective spin-labeled residues S39, S46, S56, S83, and C150 with a spring force of 30 kcal/(mol·Å^2^). Each round ran for 100ps with time step of 2 fs at _T_ = 300 K and _P_ = 0.978 atm. The trajectories and snapshots were saved every 1 ps. The initial/target distances for each CA atom pair were 73 Å/68 Å for S39, 57 Å /43 Å for S46, 26 Å/ 32 Å for S56, 51 Å/64 Å for S83, 12 Å/25 Å for C150, respectively. The target distances were quickly reached in the first 2 ps in each round (@fig:time-evolution). 

![Time evolution of the CA-CA distances. We consider five residue pairs, S39-S39, S46-S46, S56-S56, S83-S83, and C150-C150 in the dimer. For each pair, the CA-CA distance are plotted as a function of time during harmonic restrained MD simulation using Amber.](figures/cp12/time-evolution.jpg){#fig:time-evolution width=100%}

(2)	Using saved snapshots, we stripped the waters and used CRYSOL[@svergun1995] of ATSAS-3.0.4-2[@franke2017] to compare the simulated SAXS curve with the experiment[@shao2021]. 
(3)	If the simulated SAXS curve was in good agreement with the experiment, then we added all-atom spin label CYR1[@jo2014] to residues S39, S46, S56, S83, and C150, respectively, using the reMD Prepper module of CHARMM-GUI[@qi2020], then measured the initial distance of each spin pairs. 
(4)	If all distances of spin pairs fell in the experimentally measured range, then we moved on to the reMD simulations by using a modified version of NAMD 2[@shen2015;@qi2020] with all-atom CHARMM36m protein force field[@huang2017]. We attached 25 copies of all-atom CYR1 spin labels to residues S39, S46, S56, S83 and C150, respectively, with reMD Prepper in vacuum for saving the computational resources. During the whole reMD simulations, the N, Cα, C, and O atoms of each spin label were fixed to the corresponding atoms in the labeled residues (@fig:spin-label) using a force constant of 10 kcal/(mol·Å^2^). The force field of all-atom CYR1 spin label[@islam2015] is provided by CHARMM-GUI. Spatial overlap among the 25 copies of CYR1 spin labels was allowed by neglecting the interactions among them. Five independent all-atom reMD simulations were conducted at 303.15 K with different random number seeds using Langevin dynamics with a damping coefficient of 5 ps^–1^. Before each reMD production run, we performed minimization and equilibration, during which only sidechain atoms were relaxed, keeping the backbone atom positions with harmonic restraints of 2 kcal/(mol·Å^2^) imposed on them. For further structural refinement, we switched off the harmonic restraints imposed on the backbone atoms, and conducted production runs of reMD simulations for 2 ns with a time step of 0.5 fs. The long-range electrostatic interactions were treated with the particle mesh Ewald (PME) method[@petersen1995] and the nonbonded interactions were truncated at a 10 Å distance cutoff. The distance distributions of each spin label pair were restrained with a force constant of 100 kcal/(mol·Å^2^) toward the experimental distance distribution histograms with a bin width of 0.025 nm. The atomic coordinates of each reMD production run were saved every 1ps. Since each spin label has 25 copies, a total number of 625 distances for each spin label pair were obtained from a single snapshot of trajectories and a total of 1, 250, 000 data points were yielded for every spin label pair from one single reMD production run. Trajectory analysis and protein visualization were performed with VMD[@humphrey1996] and PyMOL[@llc2015], respectively.

![The fixation between CYR1 spin label and its attached residues. The all-atom CYR1 spin label has a main chain like amino acid. The reMD simulation adds and fixes the spin label model by overlapping the main chain of spin label attached residue and the main chain of spin labels with a harmonic force constant of 10 kcal/(mol·Å^2^).](figures/cp12/spin-label.jpg){#fig:spin-label width=100%}

## RESULTS and DISCUSSION

### AlphaFold2 Model
The top five dimer models for WT, S46C, and S56C mutants are shown in @fig:af2-output, including mostly elongated forms (WT-1, 3–5) and over-folded triangular one (WT-2). The small angle X-ray scattering revealed that the CP12 dimer takes an elongated cylindrical form with kinks[@shao2021]. Accordingly, we selected a representative WT model from the five candidates (@fig:predicted-model), where one monomer is shown in cartoon representation with each residue being colored according to its per-residue confidence score, while the other is in surface representation. Overall, the helical regions were modeled with high confidence, except for the C-terminal helices and some parts of the second helices. On the other hand, the random coils, potentially disordered regions, were modeled with low confidence. The coexistence of both globular folded and potential disordered regions is consistent with the Kratky analysis of the SAXS experiment[@shao2021]. This AF2 model shows an antiparallel dimer with a coiled-coil composed of residues 46–82 (among them 45–75 with high prediction confidence), in line with the inference that the region containing residues 46–82 has a high propensity for coiled-coil arrangement3. For this model, the estimated percentages of the secondary structure elements using YASARA View22 were 79.1% helix, 2.5% turn, 18.4% random coil, and 0% β sheet; whereas the occurrence of 10% β-sheet was reported by the measurement of circular dichroism spectra[@shao2021]. Two highly conserved AWD_VEEL motifs and two cysteine residues are highlighted with sticks and spheres, respectively. Neither inter- nor intra-molecular disulfide bridge between two cysteine residues is found in the AF2 model, which is mutually corroborated with experiment results[@shao2021].
 
![AlphaFold2 model of wild-type Thalassosira pseudonana CP12 dimer. One monomer is shown in cartoon representation using a color scheme based on confidence measure (blue: high, cyan: high medium, yellow: low medium, red: low), and the other monomer presents a translucent surface format. Two AWD_VEEL motifs and two cysteine residues (C142 and C150) are shown with sticks and spheres, respectively. ](figures/cp12/predicted-model.jpg){#fig:predicted-model width=100%}

The estimated SAXS curve based on the AF2 model was compared with the experimental data (@fig:ambermd A): The intensity in the low q region (< 0.15 Å^–1^) was in good agreement, whereas the high q region (> 0.15 Å^–1^) exhibits a poor fit. Also, the radius of gyration, Rg, (= 35.7 Å) of the AF2 model was smaller than the experimental value (Rg = 38.2 ± 0.4 Å) inferred by the SAXS measurement[@shao2021]. Furthermore, we analyzed the MTSL-MTSL distance distribution based on the AF2 model for different CP12 variants (@fig:ambermd D). There was no overlap between the prediction model and the DEER experiment for the distance distribution of S39R1 and C150R1 spin pairs, indicating limited reliability of the AF2 model, although it successfully captures the structural information on the relatively rigid parts of the protein system.

### Molecular Dynamic Simulations

![(A) Comparison of SAXS curves between AlphaFold2 model and experiment; (B) Comparison of SAXS curves between initial structure of restrained-ensemble molecular dynamics (reMD) simulation and experimental data; In both (A) and (B), an insert figure shows the SAXS curve at low q region (0 – 0.1 Å^–1^) and gives the Rg for both experiment and predicted model. (C) Initial structure of CP12 with 25 copies of R1 spin labels for the production run of reMD simulations; (D) Comparison of experimental and calculated distance distribution between the spin labels on 5 the residue pairs. The SAXS discrepancy between simulated and experimental data, $\chi$^2^, for both (A) and (B), are marked in blue text.](figures/cp12/amberMD.jpg){#fig:ambermd width=100%}

To improve the AF2 model, further refinement was conducted using restrained MD simulation techniques. The initial model for harmonic restrained MD simulations was a modified AF2 model whose two C-termini were moved away from the coiled-coil region of the dimer using PyMOL (@fig:scheme B). The harmonic restrained MD process was, then, followed by the restrained-ensemble MD (reMD) process (@fig:scheme A). The initial model for reMD simulations shows a better fit in the SAXS curve with the experimental curve than the AF2 model does (@fig:ambermd B). Hereafter we call this model the reMDini model. The estimated percentages of the secondary structure elements in the reMDini model using YASARA View[@krieger2014] were 50.6% (α-helix), 2.1% (β-sheet), 12.0% (turn), and 35.3% (random coil). The contents of α-helix and random coil were in good agreement with circular dichroism spectrum measurements, 32~50% (α-helix) and 27~48% (random coil), respectively. Notably, the C-termini of the AF2 model experienced considerable disorder, and well-folded long α-helices in the AF2 model were partially unfolded to be short α-helices in the reMDini model. Then, we attached 25 copies of the all-atom R1 spin labels to residues S39, S46, S56, S83, and C150, for both monomers in the reMDini model as shown in @fig:ambermd C, and started 5 independent reMD simulations with different random number seeds. Using these reMD trajectories, we analyzed the distance distributions of 5 spin label pairs and compared those with experimental ESR/DEER data (@fig:ambermd D). We observed that the simulated distance distributions of all spin pairs from the five different simulations were strikingly similar, indicating a good convergence for the sampling of the distance distributions in the reMD simulations. We compared the simulated distance distributions with those derived from the EPS/DEER experiment. As a result, For S39R1, S83R1, and C150R1 spin pairs, the simulated distributions are well overlapped with the experimental distributions, whereas we recognize partial overlaps for S46R1 and S56R1. We notice that the experimental distance distribution for the S56R1 spin pair is much broader than the simulated distance distribution. Also, the experimental distance distribution for the S46R1 spin pair exhibits two peaks, which are missing in the simulated distance distribution. It should be noted that partial dissociation of the dimer was reported by the ESI-MS experiment[@shao2021]. Taking these observations into account, we may conclude that the coiled-coil region of the dimer in a real solution environment is, possibly, more flexible than that observed in the simulations.

![The radius of gyration (Rg) distribution of 20, 000 sampled structures from reMD simulations.](figures/cp12/rg-distribution.jpg){#fig:rg-distribution width=70%}

For a comprehensive evaluation on the reMD sampling structures, we extracted 4, 000 structures from each parallel run of reMD and obtained 20, 000 in total as an ensemble structure pool. We performed GAJOR analysis to get the distribution of radius of gyration (Rg) for these 20, 000 sampled structures, shown in @fig:rg-distribution. Two major peaks of Rg distribution are recognized at about 38.1 and 38.9 Å, which are much closer to the experimental data (38.2 ± 0.4 Å) than AF2 predicted model (35.7 Å), suggesting a significant improvement on refinement of CP12 dimer model by using MD simulations.

## Summary
