
# Structural and dynamic characterization of CP12 protein {#cha6}

Structural analysis of intrinsically disordered regions (IDRs) within proteins is a long-standing challenge in structural biology due to their considerable flexibility and spatiotemporal heterogeneity.
In this chapter, we report a new strategy for characterizing the conformational ensemble of IDRs and its application to the _Thalassosira pseudonana_ chloroplastic protein (CP12) homodimer.
The method consists of three stages: (1) Obtaining an initial guess structure based on the amino acid sequence using AlphaFold-Multimer; (2) Harmonic-restrained MD simulations to meet the experimental SAXS and EPR/DEER data; (3) Refinement of the conformational ensemble by using restrained-ensemble MD simulations based on the information from DEER and SAXS experiments.
The simulated SAXS curve and inter-label distance distributions of our CP12 model were in good agreement with the experimental data, indicating that a combination of deep learning based algorithms like AlphaFold-Multimer, MD simulations, SAXS, and EPR/DEER experiments would provide a new possibility for structural characterization of IDRs and their complexes.

## AlphaFold2 Model
The top five dimer models for WT, S46C, and S56C mutants are shown in @fig:af2-output, including mostly elongated forms (WT-1, 3–5) and over-folded triangular one (WT-2). The small angle X-ray scattering revealed that the CP12 dimer takes an elongated cylindrical form with kinks[@shao2021]. Accordingly, we selected a representative WT model from the five candidates (@fig:predicted-model), where one monomer is shown in cartoon representation with each residue being colored according to its per-residue confidence score, while the other is in surface representation. Overall, the helical regions were modeled with high confidence, except for the C-terminal helices and some parts of the second helices. On the other hand, the random coils, potentially disordered regions, were modeled with low confidence. The coexistence of both globular folded and potential disordered regions is consistent with the Kratky analysis of the SAXS experiment[@shao2021]. This AF2 model shows an antiparallel dimer with a coiled-coil composed of residues 46–82 (among them 45–75 with high prediction confidence), in line with the inference that the region containing residues 46–82 has a high propensity for coiled-coil arrangement3. For this model, the estimated percentages of the secondary structure elements using YASARA View22 were 79.1% helix, 2.5% turn, 18.4% random coil, and 0% β sheet; whereas the occurrence of 10% β-sheet was reported by the measurement of circular dichroism spectra[@shao2021]. Two highly conserved AWD_VEEL motifs and two cysteine residues are highlighted with sticks and spheres, respectively. Neither inter- nor intra-molecular disulfide bridge between two cysteine residues is found in the AF2 model, which is mutually corroborated with experiment results[@shao2021].
 
![AlphaFold2 model of wild-type Thalassosira pseudonana CP12 dimer. One monomer is shown in cartoon representation using a color scheme based on confidence measure (blue: high, cyan: high medium, yellow: low medium, red: low), and the other monomer presents a translucent surface format. Two AWD_VEEL motifs and two cysteine residues (C142 and C150) are shown with sticks and spheres, respectively. ](figures/cp12/predicted-model.jpg){#fig:predicted-model width=100%}

The estimated SAXS curve based on the AF2 model was compared with the experimental data (@fig:ambermd A): The intensity in the low q region (< 0.15 Å^–1^) was in good agreement, whereas the high q region (> 0.15 Å^–1^) exhibits a poor fit. Also, the radius of gyration, Rg, (= 35.7 Å) of the AF2 model was smaller than the experimental value (Rg = 38.2 ± 0.4 Å) inferred by the SAXS measurement[@shao2021]. Furthermore, we analyzed the MTSL-MTSL distance distribution based on the AF2 model for different CP12 variants (@fig:ambermd D). There was no overlap between the prediction model and the DEER experiment for the distance distribution of S39R1 and C150R1 spin pairs, indicating limited reliability of the AF2 model, although it successfully captures the structural information on the relatively rigid parts of the protein system.

## Molecular Dynamic Simulations

![(A) Comparison of SAXS curves between AlphaFold2 model and experiment; (B) Comparison of SAXS curves between initial structure of restrained-ensemble molecular dynamics (reMD) simulation and experimental data; In both (A) and (B), an insert figure shows the SAXS curve at low q region (0 – 0.1 Å^–1^) and gives the Rg for both experiment and predicted model. (C) Initial structure of CP12 with 25 copies of R1 spin labels for the production run of reMD simulations; (D) Comparison of experimental and calculated distance distribution between the spin labels on 5 the residue pairs. The SAXS discrepancy between simulated and experimental data, $\chi$^2^, for both (A) and (B), are marked in blue text.](figures/cp12/amberMD.jpg){#fig:ambermd width=100%}

To improve the AF2 model, further refinement was conducted using restrained MD simulation techniques. The initial model for harmonic restrained MD simulations was a modified AF2 model whose two C-termini were moved away from the coiled-coil region of the dimer using PyMOL (@fig:scheme B). The harmonic restrained MD process was, then, followed by the restrained-ensemble MD (reMD) process (@fig:scheme A). The initial model for reMD simulations shows a better fit in the SAXS curve with the experimental curve than the AF2 model does (@fig:ambermd B). Hereafter we call this model the reMDini model. The estimated percentages of the secondary structure elements in the reMDini model using YASARA View[@krieger2014] were 50.6% (α-helix), 2.1% (β-sheet), 12.0% (turn), and 35.3% (random coil). The contents of α-helix and random coil were in good agreement with circular dichroism spectrum measurements, 32~50% (α-helix) and 27~48% (random coil), respectively. Notably, the C-termini of the AF2 model experienced considerable disorder, and well-folded long α-helices in the AF2 model were partially unfolded to be short α-helices in the reMDini model. Then, we attached 25 copies of the all-atom R1 spin labels to residues S39, S46, S56, S83, and C150, for both monomers in the reMDini model as shown in @fig:ambermd C, and started 5 independent reMD simulations with different random number seeds. Using these reMD trajectories, we analyzed the distance distributions of 5 spin label pairs and compared those with experimental ESR/DEER data (@fig:ambermd D). We observed that the simulated distance distributions of all spin pairs from the five different simulations were strikingly similar, indicating a good convergence for the sampling of the distance distributions in the reMD simulations. We compared the simulated distance distributions with those derived from the EPS/DEER experiment. As a result, For S39R1, S83R1, and C150R1 spin pairs, the simulated distributions are well overlapped with the experimental distributions, whereas we recognize partial overlaps for S46R1 and S56R1. We notice that the experimental distance distribution for the S56R1 spin pair is much broader than the simulated distance distribution. Also, the experimental distance distribution for the S46R1 spin pair exhibits two peaks, which are missing in the simulated distance distribution. It should be noted that partial dissociation of the dimer was reported by the ESI-MS experiment[@shao2021]. Taking these observations into account, we may conclude that the coiled-coil region of the dimer in a real solution environment is, possibly, more flexible than that observed in the simulations.

![The radius of gyration (Rg) distribution of 20, 000 sampled structures from reMD simulations.](figures/cp12/rg-distribution.jpg){#fig:rg-distribution width=70%}

For a comprehensive evaluation on the reMD sampling structures, we extracted 4, 000 structures from each parallel run of reMD and obtained 20, 000 in total as an ensemble structure pool. We performed GAJOR analysis to get the distribution of radius of gyration (Rg) for these 20, 000 sampled structures, shown in @fig:rg-distribution. Two major peaks of Rg distribution are recognized at about 38.1 and 38.9 Å, which are much closer to the experimental data (38.2 ± 0.4 Å) than AF2 predicted model (35.7 Å), suggesting a significant improvement on refinement of CP12 dimer model by using MD simulations.

## Summary
